website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -797,6 +829,53 @@ curl \
   https://app.terraform.io/api/v2/policy-sets/polset-3yVQZvHzf5j3WRJ1/relationships/workspaces
 ```
 
+## Attach a Policy Set to exclusions
+
+`POST /policy-sets/:id/relationships/excludedWorkspaces`
+
+| Parameter | Description                                                                                                 |
+| --------- | ----------------------------------------------------------------------------------------------------------- |
+| `:id`     | The ID of the policy set to attach to excluded workspaces. Use the "List Policy Sets" endpoint to find IDs. |
```

Comment:
The phrasing here is a little hard to follow. Is the idea that you pass Terraform Cloud the id of a policy set _and_ the workspaces it should exclude from that set? I tried rephrasing, but I'm not sure if this is accurate?
```suggestion
| `:id`     | The ID of a policy set that you want Terraform Cloud to exclude from the workspaces you specify. Use the "List Policy Sets" endpoint to find IDs. |
```

Reply:
(gentle bump on this, do you think the above suggestion works?)

Reply:
This is accurate, but the wording can be perhaps simplified?
How about this: The ID of a policy set that you want to exclude the specified workspaces from. What do you think about this?

Reply:
@rkoron007 in case you missed it, any thoughts on this?

Reply:
On second thought, I think this works.

Reply:
fixed!